scripts/cicd/slack/merge-notif.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+# vairables
+SLACK_NOTIFICATION_CHANNEL_WEBHOOK_URL=$1
+REPO_URL=$2
+REPO_DISPLAY_NAME=$3
+PR_URL=$4
+PR_TITLE=$5
```

Comment:
I think those value are repo value not PR value so using `REPO` might make more sense

Reply:
git repo name, it's repo
for pull request information it's pr value

we need both.

Reply:
@GeekEast 
When you run this script on merge action which is 
```
on:
  push:
    branches:
      - main
      - develop
```
I'm not sure it's easy to get the pull request url, please double check

Reply:
Hi @tonynguyenit18 you can see in the usage example on this page.

I actually use

```yml
on:
  pull_request:
    types:
      - closed
```

Reply:
I think for main and develop branches, we don't allow force push, right?

Reply:
@GeekEast 
For main we don't for develop we will if we follow release cycle
But the on push is not only about force push, that action even is to run when we merge code from PR to main or develop branch

Reply:
> Hi @tonynguyenit18 you can see in the usage example on this page.
> 
> I actually use
> 
> ```yaml
> on:
>   pull_request:
>     types:
>       - closed
> ```

Yes I saw that, that's why you can get the PR url, but this script is to run when code is merge to develop and main so that's event is not suitable for our usecase this script